.packit.yaml

```diff
@@ -21,3 +21,12 @@ jobs:
     - fedora-31-x86_64
     - fedora-rawhide-x86_64
   trigger: pull_request
+
+jobs:
```

Comment:
```suggestion
```
with this, you redefine jobs and thus remove copr-build :P no, packit is not taking a nap

Reply:
Oh! I see. Bad copy and paste. Thanks for spotting this.